content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md

```diff
@@ -6,6 +6,28 @@ menu:
     parent: About the project
 ---
 
+
+## v1.5.8 [unreleased]
+
+## Breaking changes
+
+- Remove support for darwin/386 builds (Go no longer supports), and add support for darwin/arm64 builds.
+
+## Features
+
+- Add InfluxDB `subscription-path` configuration option to allow Kapacitor to run behind a reverse proxy, thanks @aspring!
+- Add support for HTTP sources in `SideloadNode` configuration, thanks @jregovic!
+- Add support for correlate in the Alerta AlertNode, thanks @nermolaev!
+- Add `details` option to the OpsGenie v2 event handler; set to `true` to use the Kapacitor alert details as OpsGenie description text, thanks @JamesClonk!
+- Send data to InfluxDB compressed as `gzip` by default.
+- Add `PrimaryProperty` and `SecondaryProperty` methods to BigPanda AlertNode.
```

Comment:
Should we say we that we added the BigPanda event handler and link to new doc [here](https://github.com/influxdata/docs-v2/pull/2032/files#diff-393c4d189c7674fa7049bb544cfdd7d983f437b7853092a8ec34dd078dc1f306)?

Not clear why 
I'd like to improve the distinction/relationship between "event handlers" and "AlertNodes"

Reply:
I'm not sure what you mean here.

Reply:
@docmerlin, are the following statements true:

- We added a new BigPanda event handler. 
- _BigPanda AlertNode_ is different than _BigPanda event handler_. 
- We added the PrimaryProperty` and `SecondaryProperty`methods to the _BigPanda AlertNode_ (implies the BigPanda AlertNode already existed).
- In our [AlertNode](https://docs.influxdata.com/kapacitor/v1.5/nodes/alert_node/) doc, we can add the two new BigPanda methods to the Property methods table.

Reply:
`BigPanda AlertNode is different than BigPanda event handler.`
They are the same thing.

Reply:
Thank you, @docmerlin!
Our docs currently describe _AlertNodes_ and _event handlers_ as two separate things...what I meant by it'd be nice to improve the docs that describe this (non) relationship.